cmd/cloudFoundryDeploy.go

```diff
@@ -833,7 +559,6 @@ func cfDeploy(
 	config *cloudFoundryDeployOptions,
 	cfDeployParams []string,
 	additionalEnvironment []string,
-	postDeployAction func(command command.ExecRunner) error,
```

Comment:
why is this not needed anymore? is the command the smoke test script execution command?

Reply:
Thank you, I am also wondering why this was removed.

Reply:
Ok now I got it. Actually, that corresponds to `stopOldAppIfRunning `and hence I have removed it.